mobile/test/common/integration/client_integration_test.cc

```diff
@@ -23,9 +23,7 @@ class ClientIntegrationTest : public BaseClientIntegrationTest,
     // TODO(abeyad): Add paramaterized tests for HTTP1, HTTP2, and HTTP3.
     setUpstreamProtocol(Http::CodecType::HTTP1);
   }
-
-  void TearDown() override {
-    cleanup();
+  void TearDown() {
```

Comment:
Sadly this needs to stick around because this class inherits from 2 classes which both define tearDown. Otherwise we get an error.